cli/cmd/project/show.go

```diff
@@ -42,7 +42,10 @@ func ShowCmd(cfg *config.Config) *cobra.Command {
 				return err
 			}
 
-			cmdutil.TablePrinter(toRow(proj.Project))
+			err = ch.Printer.PrintResource(toRow(proj.Project))
```

Comment:
Can be shortened to just `return  ch.Printer.PrintResource(toRow(proj.Project))`